.ado/templates/build-rnw.yml

```diff
@@ -57,12 +62,12 @@ steps:
         /p:BaseIntDir=$(BaseIntDir)
```

Comment:
Out of scope for this PR, but I think we might be able to remove the IntDir shenanigans we were doing before. We used to have this to redirect some build targets to a different drive when we were previously running out of disk space. It has mean we have been less likely to catch regressions in built in IntDir logic (which believe it or not has broken multiple times).

Reply:
Let's do in a future PR.